src/ReactionService.Data/ReactionService.Data.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+
+  <ItemGroup>
+	<ProjectReference Include="..\ReactionService.Data.Provider.MsSql.Ef\ReactionService.Data.Provider.MsSql.Ef.csproj" />
```

Comment:
This spaces happens, 'cause you use tabs on lines 6 and 7, but spaces on 8. In VS/Rider this looks the same, but better to change this all to tabs or spaces (depends on your religion).

Reply:
I prefer spaces here, 'cause everywhere else we add sapces in csproj files

Reply:
Done